### ISSUE TITLE
Mismatched types error

### ISSUE THREAD

**Issue by hermanius** [LaurentMazare/tch-rs#175]
Hi,

when compiling tch 0.1.6 I get the following errors (inside a Ubuntu docker container running on an Nvidia Jetson).
Is there something I can do to solve this problem?

   Compiling tch v0.1.6
error[E0308]: mismatched types
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/tch-0.1.6/src/wrappers/jit.rs:113:35
    |
113 |                     ati_bool_list(v.as_ptr(), v.len() as c_int)
    |                                   ^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error[E0308]: mismatched types
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/tch-0.1.6/src/wrappers/jit.rs:181:62
    |
181 |                 unsafe_torch_err!(ati_to_bool_list(c_ivalue, c_array.as_mut_ptr(), len));
    |                                                              ^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*mut u8`
               found raw pointer `*mut i8`

error[E0308]: mismatched types
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/tch-0.1.6/src/wrappers/jit.rs:267:57
    |
267 |         let c_module = unsafe_torch_err!({ atm_load_str(buffer_ptr, buffer.len()) });
    |                                                         ^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0308`.
error: could not compile `tch`.

**Comment by LaurentMazare** [comment]
Interesting, I don't think the same issue happens on x86 platforms.
That said I just pushed this [commit](https://github.com/LaurentMazare/tch-rs/commit/fb218a15132dfe7de3712bb62a992bea92222404) that should hopefully help but I don't have a good way to check. Note that we also released the changes for PyTorch v1.5 so you will have either to update to PyTorch 1.5 or to apply the previous commit to your current branch (this should be straightforward as it's only a couple lines).
